src/w3c/templates/headers.js

```diff
@@ -5,13 +5,25 @@ import showLink from "./show-link";
 
 export default conf => {
   const html = hyperHTML;
+  const specTitleElem = document.querySelector("h1#title, body>h1") || document.createElement("h1") ;
+  if (specTitleElem.parentElement) { // remove it from the DOM tree, because we are going to relocate it
```

Comment:
We have a potential conflict here... if `conf.specTitle` is set, then `conf.specTitle` should win... and we should emit a an error about duplicate titles.